internal/pkg/cli/pipeline_init.go

```diff
@@ -42,6 +42,9 @@ import (
 )
 
 const (
+	fmtPipelineInitNamePrompt  = "What would you like to %s this pipeline?"
```

Comment:
Nit: I'm seeing this enthusiastic "**name**" in `app init` but not in the other init commands. I think we should be consistent. I personally could do without it; what do you think?

Reply:
This is a good call out....I've been looking around and it's looks like we're pretty inconsistent around this point. A few examples of highlighting that I found:
- [secret init](https://github.com/dannyrandall/copilot-cli/blob/baf44a7c3578efd5167a855a9a7a3faa39ece6bf/internal/pkg/cli/secret_init.go#L391)
- [job init](https://github.com/dannyrandall/copilot-cli/blob/0ca8e4d5638053867ae7c3cf7a2b1266a3e0ee75/internal/pkg/cli/job_init.go#L36)

So I'll leave it as is for now, but it's an inconsistency we should probably make a decision on and fix it to be the same everywhere.

Reply:
I dig it 😄 the original idea was to have this bold term also becoming `prompt.WithFinalMessage`